src/AspNet.Security.OAuth.QQ/QQAuthenticationHandler.cs

```diff
@@ -38,19 +36,23 @@ public class QQAuthenticationHandler : OAuthHandler<QQAuthenticationOptions>
             [NotNull] AuthenticationProperties properties,
             [NotNull] OAuthTokenResponse tokens)
         {
-            string identifier = await GetUserIdentifierAsync(tokens);
-            if (string.IsNullOrEmpty(identifier))
+            var (errorCode, openid, unionid) = await GetUserIdentifierAsync(tokens);
```

Comment:
Please refactor like the below as it makes it easier to read.

```suggestion
            (int errorCode, string openid, string unionid) = await GetUserIdentifierAsync(tokens);
```